apps/heureka/src/components/Services/ServicesList/ServicesList.tsx

```diff
@@ -64,7 +64,7 @@ export const ServicesList = ({ filterSettings }: ServiceListProps) => {
         </DataGrid>
       </div>
       {!!totalNumberOfPages && (
-        <div className="flex justify-center">
+        <div className="flex justify-end">
```

Comment:
Why not use the here the juno Stack component?

Reply:
This will go away later because we'll be having infinite scroll.